INSTALL.md

```diff
@@ -202,6 +202,81 @@ Possible sources of errors:
   and [compileBLAS.cmd](https://github.com/AMICI-dev/AMICI/blob/master/scripts/compileBLAS.cmd)
   may serve as guidance on how to install openBLAS using msvc.
 
+### Windows using MSVC (Visual Studio)
+
+#### Visual Studio
+We assume that Visual Studio (not to be confused with Visual Studio Code) is already installed. Using Visual Studio Installer, the following components need to be included:
+
+* Microsoft Visual C++ (MSVC). This is part of multiple packages, including Desktop Development with C++.
+* Windows Universal C Runtime. This is an individual component and installs some DLLs that we need.
+
+#### openBLAS
+To install open BLAS, download the following scripts from AMICI:
+
+https://github.com/AMICI-dev/AMICI/blob/master/scripts/installOpenBLAS.ps1
+https://github.com/AMICI-dev/AMICI/blob/master/scripts/compileBLAS.cmd
+
+The first script needs to be called in Powershell, and it needs to call `compileBLAS.cmd`, so you will need to modify line 11:
+
+    C:\Users\travis\build\AMICI\scripts\compileBLAS.cmd
+
+so that it matches your directory structure.
+This will download openBLAS and compile it, creating
+
+    C:\BLAS\lib\openblas.lib
+    C:\BLAS\bin\openblas.dll
+
+You will also need to define two environment variables:
+
+    BLAS_LIBS="/LIBPATH:C:\BLAS\lib openblas.lib"
+    BLAS_CFLAGS="/IC:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10"
+    
+One way to do that is to run a PowerShell script with the following commands:
+
+    [System.Environment]::SetEnvironmentVariable("BLAS_LIBS", "/LIBPATH:C:\BLAS\lib openblas.lib", [System.EnvironmentVariableTarget]::User)
+    [System.Environment]::SetEnvironmentVariable("BLAS_LIBS", "/LIBPATH:C:\BLAS\lib openblas.lib", [System.EnvironmentVariableTarget]::Process)
+    [System.Environment]::SetEnvironmentVariable("BLAS_CFLAGS", "-IC:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10", [System.EnvironmentVariableTarget]::User)
+    [System.Environment]::SetEnvironmentVariable("BLAS_CFLAGS", "-IC:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10", [System.EnvironmentVariableTarget]::Process)
+
+The call ending in `::Process` sets the environment variable in the current process, and it is no longer in effect in the next process. The call ending in `::User` is permanent, and takes effect the next time the user logs on.
```

Comment:
```suggestion
The call ending in `Process` sets the environment variable in the current process, and it is no longer in effect in the next process. The call ending in `User` is permanent, and takes effect the next time the user logs on.
```

(just because this seems to cause trouble with doxygen)